marlowe-test/src/Spec/Marlowe/Semantics/Arbitrary.hs

```diff
@@ -721,43 +701,49 @@ instance Arbitrary (Value Observation) where
   shrink (ChoiceValue c)      = ChoiceValue <$> shrink c
   shrink (UseValue v)         = UseValue <$> shrink v
   shrink (Cond o x y)         = [Cond o' x y | o' <- shrink o] ++ [Cond o x' y | x' <- shrink x] ++ [Cond o x y' | y' <- shrink y]
-  shrink _                    = []
+  shrink (Constant _)         = []
+  shrink TimeIntervalStart    = []
+  shrink TimeIntervalEnd      = []
```

Comment:
Expand wildcard case to explicit cases (so new cases must be handled when added).